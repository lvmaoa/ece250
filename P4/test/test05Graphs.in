i 1 2 1
i 2 3 1
i 2 4 1
i 4 1 1
mst 1
mst 4
i 1 5 1
mst 1
mst 4
i 2 6 1
mst 1
mst 2
exit
