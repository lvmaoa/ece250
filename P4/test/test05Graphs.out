success
success
success
success
4
4
success
5
5
success
6
6
